cni/network/network_windows.go

```diff
@@ -55,7 +56,8 @@ func handleConsecutiveAdd(containerId, endpointId string, nwInfo *network.Networ
 		return result, nil
 	}
 
-	return nil, nil
+	err = fmt.Errorf("endpoint %v not found", endpointId)
```

Comment:
Try to return the error returned from the platform instead of constructing a new one.

Reply:
done.